less/navbar.less

```diff
@@ -11,19 +11,21 @@
   overflow: visible;
   margin-bottom: @baseLineHeight;
 
+  &.navbar-no-margin-bottom {
+    margin-bottom: initial;
```

Comment:
FYI:

![navbar](https://cloud.githubusercontent.com/assets/1792593/15959006/80d52988-2ef0-11e6-8ec2-856699dd5294.PNG)

No explanation as to why